src/mainwindow.cpp

```diff
@@ -100,6 +112,85 @@ void MainWindow::updateBoms(DbManager *d)
     }
 }
 
+bool MainWindow::refillMatchFromSaveFileToDatabase(const std::string& path)
+{
+    // Read and stock all clubs
+    std::ifstream matchFlow(path.c_str(), std::ios::in);
+    if (!matchFlow)
+    {
+        std::cerr << "Failed to open Matchs.txt for reading!" << std::endl;
+        return false;
+    }
+
+    int firstScore, secondScore;
+    std::string firstTeam, secondTeam;
+    int ind1(1), ind2(1);
+
+    std::string line;
+    while (getline(matchFlow, line, '\n'))
+    {
+        std::istringstream iss(line);
+        std::string aString;
+        int count(-1);
+        while (getline(iss, aString, '/'))
+        {
+            count++;
+            if (count == 0)
+                continue;
+
+            if (count == 1)
+                continue;
+
+            switch (count % 4)
+            {
+                case 2:
+                    firstTeam   =  aString; break;
+                case 3:
+                    firstScore  = atoi(aString.c_str()); break;
+                case 0:
+                    secondScore = atoi(aString.c_str()); break;
+                case 1:
+                    secondTeam  =  aString; break;
+            }
+
+            if ((count % 4) == 1)
+            {
+                unsigned int firstTeamInd, secondTeamInd;
+                for (unsigned int i = 0; i < aTeamsId.size(); ++i)
+                {
+                    if (aTeamsId[i]->getSmallName().toStdString() == firstTeam)
+                        firstTeamInd = i+1;
+                    else if (aTeamsId[i]->getSmallName().toStdString() == secondTeam)
+                        secondTeamInd = i+1;
+                }
+
+                std::cout << std::setw(3) << ind1++ << " :"
+                          << std::setw(4) << firstTeam << " (" << std::setw(2) << firstTeamInd;
+                std::cout << ") - ";
+                std::cout << std::setw(4) << secondTeam << " (" << std::setw(2) << secondTeamInd;
+                std::cout << ") / " << firstScore << " - " << secondScore << std::endl; 
+            
+                if (apDbConnection->isOpen())
+                {
+                    if (!apDbConnection->fillMatch(ind2++, firstTeamInd, secondTeamInd, firstScore, secondScore))
+                        std::cout << "Oups..." << std::endl;
+                }
+            }
+        }
+    }
+
+    matchFlow.close();
+
+    return true;
+
+    /*
+    for (unsigned int i = 1; i < aMatchesId.size(); ++i)
+    {
+        
+    }
+    */
```

Comment:
Do not forget to remove the unwanted commented section ;)

Reply:
Sorry!